backend/open_webui/models/chats.py

```
@@ -458,65 +933,167 @@ def get_chat_list_by_chat_ids(
+        """
+        try:
+            with get_db_context(db) as db:
+                result = (
+                    db.query(Chat.folder_id).filter_by(id=id, user_id=user_id).first()
+                )
+                return result[0] if result else None
+        except Exception:
+            return None
+
+    def get_chats(
+        self, skip: int = 0, limit: int = 50, db: Optional[Session] = None
```

Ruff (0.15.4)

[warning] Unused method argument: `skip`

(ARG002)